### PR TITLE
add es6 entry point

### PR DESCRIPTION
**Pull request by capaj**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "2.0.5",
   "description": "React component for truncating multi-line spans and adding an ellipsis",
   "main": "lib/Truncate.js",
+  "module": "src/Truncate.js",
   "files": [
     "lib"
```
Comment (PabloSichert) [inline package.json]: You need to include `"src"` here as well, otherwise it won't be included in the npm package

**Comment by PabloSichert** [comment]
Could you reference any source that's explaining what the `"module"` key does in a `package.json` and how it's supposed to be used?

**Comment by capaj** [comment]
@PabloSichert of course: https://github.com/rollup/rollup/wiki/pkg.module

I'll add that src dir, forgot about that, sorry.

**Comment by PabloSichert** [comment]
I am a bit hesitant of this change as is, because

- "[...] `module` is more likely to become standardized.", so not a standard yet
- "Wait, it just means import and export – not other future JavaScript features? [...] you should transpile them if you want your library to work in environments that don't support those features."

The first one can be excused, as the additional key does not harm any compatibility. But the transpilation should be put in place.

It would be better practice to leave `"src"` out, but generate an additional bundle in `"lib"`.

**Comment by PabloSichert** [comment]
Released on npm with [`v2.1.0`](https://github.com/One-com/react-truncate/releases/tag/v2.1.0).

**Pull request merged**